justfile

```diff
@@ -55,7 +55,7 @@ deploy:
     @just up
 
 # Tear down the database, remove the volumes, recreate the database, and populate it with sample data
-fresh-start:
+fresh-start: dotenv
```

Comment:
Is fresh-start used in prod deployments? I'm worried that someone could run it without defining a .env file first and end up with insecure defaults

Reply:
It is not and should not be used in prod environments!